routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
```

Comment:
Simplify your code, you're not reusing this anywhere

Reply:
fixed da870f748e6ce67fb3db6e899a29fef8a6e767d6